pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -455,3 +467,16 @@ func MakeRelative(path, base string) (string, error) {
 	}
 	return path, nil
 }
+
+// HomeDir returns the home directory for the current user
+func HomeDir() string {
```

Comment:
Mostly public because we also use it downstream. Moved to an appropriate package under `util`.